src/auth0-session/get-config.ts

```diff
@@ -164,7 +164,7 @@ export const get = (params: ConfigParameters = {}): Config => {
     ...params
   };
 
-  const { value, error, warning } = paramsSchema.validate(config);
+  const { value, error, warning } = paramsSchema.validate(config, { allowUnknown: true });
```

Comment:
Otherwise the validation would fail because of the new extra fields.

Reply:
The issue with that approach is that you can't really have a configuration object that is any different from the one the underlying session library uses. That's not flexible and is not really an option, as it's a hard coupling with that package. I don't think that being able to catch misspellings is worth trading off for that flexibility.

Reply:
I'll come up with a solution for this and https://github.com/auth0/nextjs-auth0/pull/253#discussion_r563631152 and share it with you

Reply:
I've put a branch here https://github.com/auth0/nextjs-auth0/tree/split-config

It should make it easier for you to add config that isn't used by the base `auth0-session` layer https://github.com/auth0/nextjs-auth0/blob/split-config/src/config.ts#L460 - I'll add a few more tests to the config module and raise a PR